.github/copilot-instructions.md

```diff
@@ -0,0 +1,1080 @@
+# Prompt for GitHub Copilot
+
```

Comment:
Given that this whole file is injected into the AI, we don't need this. (And I realize that messes with all other headings in the file… )

```suggestion
```